portfolio/src/main/webapp/index.html

```diff
@@ -12,9 +12,11 @@
         <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
         <script src="script.js"></script>
     </head>
-    <body>
+    <body onload="FetchDispGreeting()">
         <!-- HEADER -->
         <header>
+            <div id="greeting">
+	</div>
```

Comment:
super nit: align this with the indentation of the open tag (or put it on the same line)